tests/modules/geometry-calculation.test.js

```diff
@@ -1,10 +1,14 @@
 const { calculateDistance, calculateJunctionPoint, isPointOnLine } = require('../../modules/geometry-calculation')
 const Line = require('../../modules/ecs6-class/line')
 const Point = require('../../modules/ecs6-class/point')
+const { getPointByX, calculateSlope, calculateNOfLineFunction } = require('../../modules/ecs6-class/line')
```

Comment:
How are they exported? 
They don't need to be exported
Learn more about mock ES6 class with jest